README.rst

```diff
@@ -114,6 +114,18 @@ New releases are frequent so always make sure that you have the latest version:
 
   pip install --upgrade pingouin
 
+Development
+-----------
+
+To build and install from source, clone this repository or download the source archive and decompress the files
+
+.. code-block:: shell
+
+  cd pingouin
+  python -m build  # optional, build a wheel
+  pytest           # test the package
+  pip install .    # install the package
```

Comment:
Can you add one line to show how install in editable mode (`pip install --editable .`)

Reply:
:+1: